### ISSUE TITLE
Overlay windows usability decrease 1.19 --> 1.21

### ISSUE THREAD

**Issue by serge2016** [MultiQC/MultiQC#2420]
### Description of bug

1. The previous version of the overlay window was easier to view and understand, numbers in the window were in columns, percents existed, there were no comma signs, window was less transparent...
1.19:
![image](https://github.com/MultiQC/MultiQC/assets/18323865/1115adc2-09fb-4bd7-858f-f2625d62d98f)

1.21:
![image](https://github.com/MultiQC/MultiQC/assets/18323865/d2b39f48-b52e-4510-a94d-300019c19e10)

2. The previous version of the overlay label had something like an arrow, pointing to a necessary line; the window itself was in the end of a line, percents existed, there were no comma signs...
1.19:
![image](https://github.com/MultiQC/MultiQC/assets/18323865/b517939a-72f4-4bc2-97a0-ce92cc36a222)

1.21:
![image](https://github.com/MultiQC/MultiQC/assets/18323865/9b271805-e0c0-43b9-a866-9a09fd3d9ddc)

### Before submitting

- [X] I have read the [troubleshooting documentation](https://multiqc.info/docs/usage/troubleshooting/).
- [X] I am using the latest release of MultiQC.
- [X] I have included a full MultiQC log, not truncated.
- [X] I have attached an input file (**.zip** if necessary) that triggers the error.

**Comment by vladsavelyev** [comment]
Yeah, that's one of those things that were easy to customize in Highcharts and very difficult with Plotly. That's on my list though!
